ngx_http_access_token_to_jwt.c

```diff
@@ -127,6 +137,20 @@ static ngx_int_t ngx_http_access_token_to_jwt_handler(ngx_http_request_t *reques
         return NGX_DECLINED;
     }
 
+    if (module_location_config->client_secret.len == 0)
+    {
+        ngx_log_error(NGX_LOG_WARN, request->connection->log, 0,
+            "Module not configured properly: missing client secret");
+        return NGX_DECLINED;
+    }
+
+    if (module_location_config->client_id.len == 0)
+    {
+        ngx_log_error(NGX_LOG_WARN, request->connection->log, 0,
+                           "Module not configured properly: missing client id");
+        return NGX_DECLINED;
+    }
+
     ngx_str_t encoded_client_credentials = module_location_config->base64encoded_client_credentials;
 
     if (encoded_client_credentials.len == 0)
```

Comment:
Even for empty configuration on client_id and client_secret this is not empty ( has the value `Og==` which is the base64 encoded `:`). I ll remove this check